twilio/rest/__init__.py

```diff
@@ -1,5 +1,7 @@
 from .base import set_twilio_proxy
 from .client import TwilioRestClient
+from .conversations import TwilioConversationsClient
```

Comment:
yes this makes importing TwilioConversationsClient, TwilioIpMessagingClient.. simpler. 
I will add these two to pyflakes list.